CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Traffic Ops now uses a consistent format for audit logs across all Go endpoints
 
 ### Changed
+- Traffic Router:  TR will now allow *STEERING target DSs to have RGB enabled. (fixes #3910)
```

Comment:
It looks like the `*` here is making the rest of this file italic. Wrapping it in backticks might fix that

Reply:
Fixed and made the entry more accurate.